lib/class-wp-rest-menu-items-controller.php

```diff
@@ -399,11 +458,11 @@ protected function prepare_item_for_database( $request ) {
 				if ( ! is_nav_menu_item( $prepared_nav_item['menu-item-parent-id'] ) ) {
 					return new WP_Error( 'invalid_menu_item_parent', __( 'Invalid menu item parent.' ), array( 'status' => 400 ) );
 				}
-				if ( $menu_item_ids && ! in_array( $prepared_nav_item['menu-item-parent-id'], $menu_item_ids, true ) ) {
+				if ( ! $menu_item_ids || ! in_array( $prepared_nav_item['menu-item-parent-id'], $menu_item_ids, true ) ) {
 					return new WP_Error( 'invalid_item_parent', __( 'Invalid menu item parent.' ), array( 'status' => 400 ) );
 				}
 			}
-		}
+		}//end if
```

Comment:
Ditto as above.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/071f64f3c82084544f6050f636b081eae460b344